Makefile

```diff
@@ -85,9 +85,12 @@ endif
 	mkdir -p build
 	touch $@
 
-oldtest: | nvim
+oldtest: | nvim tags
```

Comment:
Well, this is a legacy test, so helpers isn't available.  If the CI scripts weren't duplicating Makefile's knowledge, this should work.